kubejs/server_scripts/enigmatica/kubejs/base/tags/blocks/ftbultimine/menril_logs.js

```diff
@@ -1,3 +1 @@
-onEvent('block.tags', (event) => {
-    event.add('ftbultimine:menril_logs', ['integrateddynamics:menril_log', 'integrateddynamics:menril_log_filled']);
-});
+//todo remove in 1.3.0
```

Comment:
This should be `//TODO: Remove in 2.0.0`, only major version bumps are breaking now :)